_layouts/glossary-rtl.html

```diff
@@ -0,0 +1,71 @@
+---
+---
+    {%- if page.title -%} 
+        {%- assign the_title = page.title -%} 
+    {%- else -%} 
+        {%- assign language = page.permalink | replace: '/', '' %}
+        {%- assign the_title = site.languages | where: "key", language | map: "title" | first -%} 
+        {%- assign the_name = site.languages | where: "key", language | map: "name" | first -%}
+    {%- endif -%} 
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+<head>
+    <title>{{site.title}}</title>
+    <link href="{{ '/static/site.css' | relative_url }}" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" type="text/css" href="{{site.baseurl}}/assets/css/glossary.css" />
+    <link rel="stylesheet" type="text/css" href="{{ relative_root_path }}/assets/css/glossary.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+<main>
+    <!-- {%- if page.direction -%}
+        {%- assign direction = page.direction -%}
+    {%- else -%}
+        {%- assign direction = 'ltr' %}
+    {%- endif -%} -->
+<head>
+    <a href="{{ site.github.repository_url }}" class="button-rtl"><small>View project on</small>GitHub</a>
+    <div class="row" dir="auto" style="margin-right: 1cm">
+        <div class="left-rtl">
+            <a href="{{'/' | relative_url}}">
+                <img src="{{'/static/parrot.svg' | relative_url}}"
+                     alt="Parrot logo"
+                     width="85%" />
+            </a>
+        </div>
+        <div class="right-rtl" style="font-family: Arial, Helvetica, sansserif; color: white; margin-right: 1cm">
+            <h1>{{the_title}}</h1>
+            <h2>{{ the_name | default: site.description }}</h2>
+        </div>
+    </div>
+</head>
+    <!--div style="padding: Opx 50px 0px 100px; width: 70%; margin: Opx 50px 0px 100px; align-content: center"-->
```

Comment:
should this commented out code be removed?

Reply:
Yeah.